### PR TITLE
snap: fixes desktop file that'll attached to the window

### PR DESCRIPTION
**Pull request by soumyaDghosh**
Similar to https://github.com/canonical/firefox-snap/pull/74

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 name: thunderbird
+title: Thunderbird
```
Comment (aleasto) [inline snapcraft.yaml]: This seems unrelated

**Comment by aleasto** [comment]
Makes sense to me!

File: snapcraft.yaml (hunk 1 above)
Comment (soumyaDghosh) [reply snapcraft.yaml]: This adds some metadata... Which I think firefox should've it too

**Comment by hellsworth** [comment]
looks great and thanks for fixing this! @seb128 can we please merge this since you have the permissions?

**Review by seb128 — CHANGES_REQUESTED** [review]
It might make sense to add a title but that isn't related to fixing the bug described there and not what the commit message describe so please revert that part and submit another commit/PR for it with rational if you want to see that landed

**Comment by soumyaDghosh** [comment]
> It might make sense to add a title but that isn't related to fixing the bug described there and not what the commit message describe so please revert that part and submit another commit/PR for it with rational if you want to see that landed

Sure... Thanks!

**Comment by soumyaDghosh** [comment]
Sorry @seb128 I forgot about this PR

**Review by seb128 — APPROVED** [review]
Thanks

**Pull request merged**